gst-play/gst-play.c

```diff
@@ -195,9 +195,12 @@ static void
 print_all_stream_info (GstPlayerMediaInfo * media_info)
 {
   guint count = 0;
-  GList *l, *list;
+  GList *list = NULL, *l;
```

Comment:
No need to initialize it with NULL here. gst_player_media_info_get_stream_list() will always return something